src/EditorFeatures/Core/Implementation/InlineRename/CommandHandlers/RenameCommandHandler.cs

```diff
@@ -7,11 +7,12 @@
 using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Operations;
+using Microsoft.VisualStudio.Utilities;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.InlineRename
 {
-    [ExportCommandHandler(PredefinedCommandHandlerNames.Rename,
-       ContentTypeNames.RoslynContentType)]
+    [ExportCommandHandler(PredefinedCommandHandlerNames.Rename, ContentTypeNames.RoslynContentType)]
+    [Order(Before = PredefinedCommandHandlerNames.Commit)]
```

Comment:
A comment explaining the reason why the ordering is necessary might be wise. (I have a pretty good guess, but not everybody will.)